onnxruntime/core/providers/dml/OperatorAuthorHelper/OperatorHelper.cpp

```diff
@@ -2480,6 +2481,50 @@ namespace OperatorHelper
                 MLOperatorTensor outputSizesTensor = kernelInformation.GetConstantInputTensor(3);
                 ReadCpuLocalTensorIntoInt32(outputSizesTensor, /*out*/ outputSizes);
             }
+
+            axes = kernelInformation.GetAttributes().GetOptionalAttributeVectorInt32(AttrName::Axes);
+            // Handle possible axes input
+            if (opsetVersion >= 18 && !axes.empty())
+            {
+                uint32_t dimCount = gsl::narrow_cast<uint32_t>(m_inputDimensions.size());
+                HandleEmptyAxes(axes, m_inputDimensions, false);
+                HandleNegativeAxes(axes, dimCount);
```

Comment:
Just for mutability clarity for future readers, could you add the typical /*out*/ comment?
See line 1354 above.

Reply:
Can you explain more what you mean? As in something like "Previous opsets didn't have an axis attribute" ? There's the comment on line 2486 that mentions possible axis input"

Reply:
Explained via Teams. Resolving.